scripts/add-event.js

```diff
@@ -29,45 +28,40 @@ const price = argv[5];
 const collective_price = argv[6];
 const name = argv[7];
 const description = argv[8];
+const username = argv[9];
+const password = argv[10];
 
-read({ prompt: "Username:" }, function (err, username) {
-  if (err) {
-    console.error("Error obtaining username:", err);
-    process.exit(1);
-  }
-  read({ prompt: "Password:", silent: true }, function (err, password) {
-    if (err) {
-      console.error("Error obtaining password:", err);
-      process.exit(1);
-    }
```

Comment:
The benefit of reading these credentials in an interactive way like this is that they then won't be logged in the shell's command history, which could pose a security risk.